virtual_rainforest/models/animals/model_schema.json

```diff
@@ -22,12 +22,27 @@
                             "diet": {
                                 "type": "string",
                                 "default": ""
+                            },
+                            "metabolic_type": {
+                                "type": "string",
+                                "default": ""
+                            },
```

Comment:
What are the possible values for metabolic type? What does a type of `""` mean?

Reply:
The possible values are "endothermic" and "ectothermic". None of these functional group properties should have default values.